plugins/GSdx/GSHwHack.cpp

```diff
@@ -85,22 +85,18 @@ bool GSC_DBZBT3(const GSFrameInfo& fi, int& skip)
 		}
 		else if(fi.TME && (fi.FBP == 0x00000 || fi.FBP == 0x00e00 || fi.FBP == 0x01000) && fi.FPSM == PSM_PSMCT32 && fi.TPSM == PSM_PSMT8H)
 		{
-			if (Dx_only) { // Ought to be fine with blending accuracy (fbmask?)
+			if (Aggressive) { // Partially works. A small black line shakes at the bottom on dx9. Can be solved with x y offset or removing outlines with Aggressive crc.
```

Comment:
`dx9` -> Direct3D 9

`x y offset` -> TC Offset x & y (does it need the x value though?)